internal/kubernetes/channelHandler.go

```diff
@@ -274,10 +276,50 @@ func (c *channelHandler) OnWindow(_ uint64, columns uint32, rows uint32, _ uint3
 	return c.exec.resize(ctx, uint(rows), uint(columns))
 }
 
+func (c *channelHandler) OnX11Request(
+	requestID uint64,
+	singleConnection bool,
+	proto string,
+	cookie string,
+	screen uint32,
+	reverseHandler sshserver.ReverseForward,
+) error {
+	if c.x11 {
+		return fmt.Errorf("X11 forwarding already setup for this channel")
+	}
+
+	c.networkHandler.logger.Debug("Setting X11 vars")
+	c.env["DISPLAY"] = "localhost:10"
+	c.env["XAUTHORITY"] = "/tmp/.Xauthority"
```

Comment:
We should think about making the location of the .Xauthority file configurable purely because some security profiles place severe restrictions on the `/tmp` directory.

Reply:
Now I'm passing along the full environment so it can be modified by configuration server variables.